src/main/java/org/openrewrite/java/spring/boot3/RemoveConstructorBindingAnnotation.java

```diff
@@ -19,8 +19,10 @@
 import org.openrewrite.Recipe;
 import org.openrewrite.internal.ListUtils;
 import org.openrewrite.java.JavaIsoVisitor;
-import org.openrewrite.java.tree.J;
-import org.openrewrite.java.tree.TypeUtils;
+import org.openrewrite.java.tree.*;
+import org.openrewrite.marker.Markers;
+
+import java.util.*;
```

Comment:
I'd add a `getSingleSourceApplicableTest` to limit the recipe sources that contain the annotations:
```java
    @Nullable
    @Override
    protected TreeVisitor<?, ExecutionContext> getSingleSourceApplicableTest() {
        return new JavaIsoVisitor<ExecutionContext>() {
            @Override
            public J.CompilationUnit visitCompilationUnit(J.CompilationUnit cu, ExecutionContext executionContext) {
                doAfterVisit(new UsesType<>("org.springframework.boot.context.properties.ConstructorBinding"));
                doAfterVisit(new UsesType<>("org.springframework.boot.context.properties.ConfigurationProperties"));
                return cu;
            }
        };
    }
```